pynos/versions/base/interface.py

```diff
@@ -421,10 +420,10 @@ def ip_address(self, **kwargs):
         method_name = None
         method_class = self._interface
         if ipaddress.version == 4:
```

Comment:
probably best to have a constant defined for any literals - same with the 6 below.